util/show-codecov.BAT

```diff
@@ -18,7 +18,7 @@ call cargo clean
 
 set CARGO_INCREMENTAL=0
 set "RUSTC_WRAPPER="    &@REM ## NOTE: RUSTC_WRAPPER=='sccache' breaks code coverage calculations (uu_*.gcno files are not created during build)
-set "RUSTFLAGS=-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off -Zno-landing-pads"
+set "RUSTFLAGS=-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off"

```

Comment:
```suggestion
set "RUSTFLAGS=-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off  -Zno-landing-pads"
```
- `leave unchanged`

I'm fixing the code coverage bug via a different method (pinning the nightly) so that the chance of coverage breaking (especially for already known-working builds) in the future is minimized.

We have to stay dependent on nightly at the moment because of lack of stabilized support for coverage in the rust toolchain. Since, by definition, it is allowed to break compatibility, pinning is needed.

I'll push the PR for it this evening ...

Reply:
ok, let me revert this change then :)